src/components/icon/iconsets/common-icons.ts

```diff
@@ -189,6 +189,7 @@ export const commonIcons = [
     'information',
     'inherit-filled',
     'inherit',
+    'input-to-process',
```

Comment:
I forgot this file exists too, @ashley-hunter shouldn't this be generated during the build?

Reply:
Ashley left a comment of the ticket about this https://portal.digitalsafe.net/browse/EL-3593 and opened a pull request to fix it.